frontend/src/components/structures/Files.js

```diff
@@ -3,34 +3,38 @@ import React from 'react';
 import File from './File';
 
 function Files(props) {
-    if (props.files.length === 0) {
-        return (
-            <tbody>
-                <tr>
-                    <td>The user has no files</td>
-                </tr>
-            </tbody>
-        );
-    }
-
+  if (props.files.length === 0) {
     return (
-        <>
-            <thead>
-                <tr>
-                    <td>name</td>
-                    <td>path</td>
-                </tr>
-            </thead>
-            <tbody>
-                {Array.isArray(props.files) &&
-                    props.files.map(file => {
-                        return (
-                            <File user={props.user} key={file.id} file={file} />
-                        );
-                    })}
-            </tbody>
-        </>
+      <tbody>
+        <tr>
+          <td>The user has no files</td>
+        </tr>
+      </tbody>
     );
+  }
+
+  const flatFileList = [].concat.apply(
```

Comment:
This should go to redux.

Reply:
Should we have separate reducer for folders then :thinking:

Reply:
It seems now that you didn't add new functionality regarding file fetching, just modified it, so I would suggest modifying the existing reducer instead.